src/components/cards/experience-listing-card/ExperienceListingCard.vue

```diff
@@ -0,0 +1,221 @@
+<template>
+  <div class="experience-listing-card">
+    <div class="container">
+      <div class="image">
+        <img :src="image" />
+      </div>
+
+      <div class="body">
+        <p
+          :class="['status', isListed ? 'isListed' : '']"
+          v-if="deviceSize <= deviceSizes.s"
+        >{{ checkListed }}</p>
+        <p class="title">{{ title }}</p>
+        <p class="country" v-if="deviceSize > deviceSizes.s">{{ country }}</p>
+        <div class="actions" v-if="deviceSize <= deviceSizes.s">
+          <div>
```

Comment:
It's alway a good practise to have class on element because you can easily understand why this bloc is there
(if possible of course)

Reply:
Done